yi-core/src/Yi/Keymap/Keys.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE FlexibleContexts  #-}
+{-# LANGUAGE LambdaCase #-}
```

Comment:
This seems unnecessary to me, is that true?

Reply:
Good point! Was for an earlier change.